NorimaConsulting/training-and-onboarding
# REC-30: user login, user logout, and user profile button components

- Added a user login button that routes to '/login' when user is logged out.
- Added a dropdown selection when user is logged is that allows:
        1. user to log out and routes to '/logout'
        2. user to view their profile and routes to '/profile'

Testing:
1. Go to /webapp directory in terminal and run 'pnpm dev'
2. In app.tsx at line 27, set 'isLoggedIn' to true to simulate user logged in, and false to simulate user logged out
2. Go to http://localhost:3000/test
3. Click the button/dropdown and you should be routed to the correct page
